test/integration/scheduler/extender_test.go

```diff
@@ -350,7 +351,8 @@ func TestSchedulerExtender(t *testing.T) {
 	}
 	policy.APIVersion = "v1"
 
-	testCtx = testutils.InitTestScheduler(t, testCtx, false, &policy)
```

Comment:
Revert this line. It looks like this is our only guard against a bug when using policy.